applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp

```diff
@@ -20,157 +20,103 @@
 
 namespace Kratos
 {
-template<SizeType TDim, class TVarType>
-ComputeHessianSolMetricProcess<TDim, TVarType>::ComputeHessianSolMetricProcess(
-        ModelPart& rThisModelPart,
-        TVarType& rVariable,
-        Parameters ThisParameters
-        ):mThisModelPart(rThisModelPart),
-          mVariable(rVariable)
+ComputeHessianSolMetricProcess::ComputeHessianSolMetricProcess(
+    ModelPart& rThisModelPart,
+    Variable<double>& rVariable,
+    Parameters ThisParameters
+    ):mThisModelPart(rThisModelPart)
 {
-    Parameters default_parameters = Parameters(R"(
-    {
-        "minimal_size"                        : 0.1,
-        "maximal_size"                        : 10.0,
-        "enforce_current"                     : true,
-        "hessian_strategy_parameters":
-        {
-            "metric_variable"                  : ["DISTANCE"],
-            "estimate_interpolation_error"         : false,
-            "interpolation_error"                  : 1.0e-6,
-            "mesh_dependent_constant"              : 0.28125
-        },
-        "anisotropy_remeshing"                : true,
-        "anisotropy_parameters":
-        {
-            "reference_variable_name"              : "DISTANCE",
-            "hmin_over_hmax_anisotropic_ratio"     : 1.0,
-            "boundary_layer_max_distance"          : 1.0,
-            "interpolation"                        : "Linear"
-        }
-    })" );
+    // We push the list of double variables
+    mrOriginVariableDoubleList.push_back(&rVariable);
 
+    // We check the parameters
+    Parameters default_parameters = GetDefaultParameters();
```

Comment:
const

Reply:
RecursivelyValidateAndAssignDefaults can not work with const Parameters

Reply:
indeed, I checked, even though I think it should be 
=> maybe after #58